core/domain/suggestion_services.py

```diff
@@ -362,6 +363,9 @@ def accept_suggestion(
                 suggestion.suggestion_id)
         )
 
+    if suggestion.edited_by_reviewer:
+        commit_message = commit_message + ' (with edits)'
+
```

Comment:
Used the flag to identify whether the suggestion is edited by the reviewer.
cc/ @seanlip